internal/utils/utils.go

```diff
@@ -17,20 +17,28 @@ package utils
 import (
 	"context"
 	"fmt"
+	"regexp"
+	"strings"
 	"time"
 
 	"github.com/cilium/cilium-cli/defaults"
 )
 
+func CheckVersion(version string) bool {
+	var check = regexp.MustCompile(`^([v]|0|[1-9][0-9]*\.)?(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-[a-zA-Z0-9]+)*\.*(0|[1-9][0-9]*)?`).MatchString
```

Comment:
This can be moved out of the function and can be created in the package scope instead. That way it doesn't have to be recreated on each call to `CheckVersion()`